server/routes/aws/onboarding.py

```
@@ -104,11 +105,11 @@ def get_aws_onboarding_links(user_id, workspace_id):
         if aws_conn and aws_conn.get('role_arn'):
             response_data["roleArn"] = aws_conn['role_arn']
         
-        logger.info(f"Retrieved AWS onboarding info for workspace {workspace_id}")
+        logger.info(f"Retrieved AWS onboarding info for workspace {sanitize(workspace_id)}")
         return jsonify(response_data)
         
     except Exception as e:
-        logger.error(f"Failed to get AWS onboarding info for workspace {workspace_id}: {e}")
+        logger.error(f"Failed to get AWS onboarding info for workspace {sanitize(workspace_id)}: {sanitize(e)}")
```

Ruff (0.15.11)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)